### PR TITLE
fix: explicitly include files in npm package

### PR DESCRIPTION
**Pull request by darscan**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

- explicitly include files in npm package (pass list) instead of ignoring files with .npmignore (block list)
- https://docs.npmjs.com/files/package.json#files

#### How should this be manually tested?

- checkout branch
- navigate elsewhere
- `npm pack path/to/branch`
- unpack resultant tarball
- `npm install package` (tarball will unpack into a folder named `package`)
- ensure `cli` works: `path/to/unpacked-package/node_modules/snyk/cli/index.js <command>`

#### Any background context you want to provide?

- we had inadvertently started bundling package binaries into the npm package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
   "main": "lib/index.js",
+  "files": [
+    "cli",
+    "help",
+    "lib",
+    "config.default.json",
+    "SECURITY.md",
+    "README.md",
+    ".snyk"
+  ],
```
Comment (adrukh) [inline package.json]: Here's what we currently publish (top level):
```
Contributor-Agreement.md
LICENSE
README.md
SECURITY.md
appveyor.yml
cli
config.default.json
docker
help
lib
node_modules
package.json
scripts
```


So how about we add `Contributor-Agreement.md` and `LICENSE` for legal terms, and `package.json` just for visibility (version value, for example)?

Comment (adrukh) [reply package.json]: `package.json` is probably good hygiene just for `npm` stuff?

Comment (darscan) [reply package.json]: Thanks :) Note: `LICENSE` cannot be excluded (along with a bunch of other files), see: https://docs.npmjs.com/files/package.json#files

Comment (darscan) [reply package.json]: I'll add `Contributor-Agreement.md`

Comment (darscan) [reply package.json]: As for `docker` and `scripts` - I don't think these should live in the published package

Comment (adrukh) [reply package.json]: Agree re `docker` and `scripts`!

Comment (darscan) [reply package.json]: Updated to add: `Contributor-Agreement.md`

**Pull request merged**